node/node_test.go

```diff
@@ -118,7 +120,7 @@ func setupFullNodes(t *testing.T, proto protocol.ConsensusVersion, verificationP
 		if err != nil {
 			panic(err)
 		}
-
+		access.Close()
```

Comment:
nice catch but shouldn't it also be done inside err branch after account.GenerateRoot?

Reply:
I've moved it before the error checking.